docs/cedarling/cedarling-quick-start.md

```
@@ -216,27 +223,34 @@ For better understanding of the TBAC flow, see the diagram below.
 ```
-@id("allow_supreme_ruler")
+@id("allow_teacher")
 permit(
-  principal in Jans::Role::"SupremeRuler",
+  principal in Jans::Role::"Teacher",
   action,
   resource
 );
 ```
 
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)